github/actions_secrets.go

```diff
@@ -16,10 +16,10 @@ type PublicKey struct {
 	Key   *string `json:"key"`
 }
 
-// GetPublicKey gets a public key that should be used for secret encryption.
+// GetRepositoryPublicKey gets a public key that should be used for secret encryption.
```

Comment:
When names get impractically long (not that this one is, but down below... and you mentioned it in the issue),
in other places in this repo we have chosen to shorten "Repository" to "Repo" and "Organization" to "Org".

I think that would work well here... but let's go through these modified files and be consistent with the renaming since this is a breaking-API-change PR anyway.

So this would be `GetRepoPublicKey`.